dbos/_sys_db.py

```diff
@@ -538,47 +543,6 @@ def get_workflow_status_within_wf(
         )
         return stat
 
-    def get_workflow_status_w_outputs(
```

Comment:
These were never actually used outside of tests, and were replaceable in those tests